arelle/CntlrCmdLine.py

```diff
@@ -325,7 +325,7 @@ def parseAndRun(args):
                              "Commands show, and module urls are '|' separated: "
                              "url specifies a plug-in by its url or filename, "
                              "relative URLs are relative to installation plug-in directory, "
-                             " (e.g., 'http://arelle.org/files/hello_web.py', 'C:\Program Files\Arelle\examples\plugin\hello_dolly.py' to load, "
+                             R" (e.g., 'http://arelle.org/files/hello_web.py', 'C:\Program Files\Arelle\examples\plugin\hello_dolly.py' to load, "
```

Comment:
It would be nice if there were a convention for `R''` vs `r''`.  (My grep might have been poor, but I don't see any instances of `R` in the repo.)

Reply:
Thanks, I was unaware of that convention.  That conventions seems reasonable to me if others agree.

Reply:
Yep seems reasonable

Reply:
This is news to me (capital R)